apps/sync/js/sync-engine/syncengine.js

```diff
@@ -195,12 +195,10 @@ uld be a Function`);
   SyncEngine.prototype = {
     _createKinto: function(kintoCredentials) {
       var kinto = new Kinto({
-        bucket: 'syncto',
-        dbPrefix: kintoCredentials.xClientState,
```

Comment:
Since https://github.com/Kinto/kinto.js/blob/master/src/collection.js#L114 includes both the `dbPrefix` and `bucket` in the database name, now that we have unique buckets per `xClientState`, we no longer have to set `dbPrefix`.